docs/userguides/console.md

```diff
@@ -0,0 +1,56 @@
+# Ape Console
+
+Ape provides an [IPython](https://ipython.readthedocs.io/) interactive console with useful pre-defined locals to interact with your project.
+
+```bash
+ape console --network ethereum:mainnet
+
+In [1]: chain.blocks.head.timestamp
+Out[1]: 1647323479
+```
+
+## Namespace Extras
+
+You can also create scripts to be included in the console namespace by adding a file (`ape_console_extras.py`) to your root project directory.  All non-internal symbols from this file will be included in the console namespace.  Internal symbols are prefixed by an underscore (`_`).
+
+An example file might look something like this:
+
+```python
+from eth_utils import encode_hex, decode_hex
+
+
+def ape_init_extras(networks):
+    return {
+        "latest": lambda key: getattr(networks.active_provider.get_block("latest"), key)
+    }
```

Comment:
Actually, in keeping with the explanation above, this doesn't mention `ape_init_extras` yet, so do it like so
```suggestion
latest = lambda key: getattr(networks.active_provider.get_block("latest"), key)
```

Reply:
Agreed